app/helpers/application_helper/toolbar/container_group_center.rb

```diff
@@ -58,6 +58,11 @@ class ApplicationHelper::Toolbar::ContainerGroupCenter < ApplicationHelper::Tool
           'pficon pficon-edit fa-lg',
           N_('Edit Tags for this #{ui_lookup(:table=>"container_group")}'),
           N_('Edit Tags')),
+        button(
+          :container_group_protect,
+          'pficon pficon-edit fa-lg',
+          N_('Manage Policies for this #{ui_lookup(:table=>"container_group")}'),
+          N_('Manage Policies')),
```

Comment:
Great.  I was wondering if I need compliance too.
But what does it mean for these entities?
Should it trigger some work, like scanning images does?
And do you think it makes sense to postpone compliance to separate PR?